charts/community-operator-crds/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: community-operator-crds
 description: MongoDB Kubernetes Community Operator - CRDs
 type: application
-version: 0.12.0
+version: 0.12.1
```

Comment:
I think it is not critical, but It is not a bug fix. Maybe worth doing 0.13.0?

Reply:
Got some background on why patch version: version of the chart is coupled with the version of the operator itself. That's why we are increasing patch version here.

I think I'm fine with it. Resolving this thread.